content/docs/spec-guides/authed-media-servers/_index.md

```diff
@@ -0,0 +1,91 @@
+metadata about the file, and the file itself (or a pointer to the file). This can
+look like either of the following two examples:
+
+```text
+Content-Type: multipart/mixed; boundary=gc0p4Jq0M2Yt08jU534c0p
+
+--gc0p4Jq0M2Yt08jU534c0p
+Content-Type: application/json
+
+{}
+
+--gc0p4Jq0M2Yt08jU534c0p
+Content-Type: text/plain
+
+This media is plain text. Maybe somebody used it as a paste bin.
+
+--gc0p4Jq0M2Yt08jU534c0p
+```
+
+```text
+Content-Type: multipart/mixed; boundary=gc0p4Jq0M2Yt08jU534c0p
+
+--gc0p4Jq0M2Yt08jU534c0p
+Content-Type: application/json
+
+{}
+
+--gc0p4Jq0M2Yt08jU534c0p
+Location: https://cdn.example.org/ab/c1/2345.txt
+
+--gc0p4Jq0M2Yt08jU534c0p
+```
+
+Note that the second example uses a Location header to point to the file instead.
+Servers will need to make a regular GET request to the provided URL to fetch the
+media - `Content-Type` and `Content-Disposition` headers will be present on that
+URL instead of in the headers on the part.
+
+Currently, the metadata object is always an empty JSON object, but in future may
+contain details about which users are able to view the media. An example in this
+area is [MSC3911].
+
+The new endpoints require authentication like most other federation endpoints.
+
+Additionally, servers will need to try the new download endpoints before falling
+back to the existing `/_matrix/media/*` ones. Specifically, if a `404 M_UNRECOGNIZED`
+error is received, servers should fall back rather than treat the media as
+non-existent.
+
+<!-- Link tree -->
+[MSC3911]: https://github.com/matrix-org/matrix-spec-proposals/pull/3911
+[MSC3916]: https://github.com/matrix-org/matrix-spec-proposals/pull/3916
+[`GET /_matrix/client/v1/media/download/{serverName}/{mediaId}`]: https://spec.matrix.org/unstable/client-server-api/#get_matrixclientv1mediadownloadservernamemediaid
+[`GET /_matrix/client/v1/media/download/{serverName}/{mediaId}/{fileName}`]: https://spec.matrix.org/unstable/client-server-api/#get_matrixclientv1mediadownloadservernamemediaidfilename
+[`GET /_matrix/client/v1/media/thumbnail/{serverName}/{mediaId}`]: https://spec.matrix.org/unstable/client-server-api/#get_matrixclientv1mediathumbnailservernamemediaid
+[`GET /_matrix/client/v1/media/config`]: https://spec.matrix.org/unstable/client-server-api/#get_matrixclientv1mediaconfig
+[`GET /_matrix/client/v1/media/preview_url`]: https://spec.matrix.org/unstable/client-server-api/#get_matrixclientv1mediapreview_url
+[`/create`]: https://spec.matrix.org/unstable/client-server-api/#post_matrixmediav1create
+[`/upload[/{serverName}/{mediaId}]`]: https://spec.matrix.org/unstable/client-server-api/#post_matrixmediav3upload
+[`GET /_matrix/federation/v1/media/download/{mediaId}`]: https://spec.matrix.org/unstable/server-server-api/#get_matrixfederationv1mediadownloadmediaid
+[`GET /_matrix/federation/v1/media/thumbnail/{mediaId}`]: https://spec.matrix.org/unstable/server-server-api/#get_matrixfederationv1mediathumbnailmediaid
```

Comment:
I plan to update all of these links to Matrix 1.11 tomorrow, when the spec goes out.